src/Workspace/Wes_Timing_System_4/_Config.lua

```diff
@@ -17,11 +17,13 @@ local config = {
 
 	-- Settings
 	-- Whether the player must be seated or not
-	PlayerMustBeSeated = true,
+	PlayerMustBeSeated = false,
 	-- How much time must be elapsed between two CCs in order for them to count twice
 	CornerCutCooldownSeconds = 1,
 	-- Whether or not a popup appears per cut, rather than once per lap
-	PopupEverycut = false,
+	PopupEverycut = true,
+	-- Shows time difference to personal best lap every sector
+	SectorTimeDelta = true,
```

Comment:
I think the default value should be false because I think the timing system should be relatively basic at its core.
```suggestion
	SectorTimeDelta = false,
```